cli/src/semgrep/commands/scan.py

```diff
@@ -276,12 +276,6 @@ def convert(
             to 3 hours.
         """,
     ),
-    optgroup.option(
-        "--core-opts",
-        hidden=True,
```

Comment:
It was hidden so perhaps it was just an internal flag? We should use `--internal-xyz` for these.